client/src/components/StarknetAgent.tsx

```diff
@@ -413,6 +444,29 @@ const StarknetAgent = () => {
 
       const formattedText = formatResponse(JSON.stringify(data));
       typeResponse({ ...newResponse, text: formattedText });
+
+      // If client uploads file we tell to the server to delete the file.
+      if (selectedFile) {
+        console.log(selectedFile.name);
+        const del = await fetch('api/key/delete', {
```

Comment:
be more explicite regarding what your are deleting here something like: delete large file